config/locales/en.yml

```diff
@@ -338,6 +338,34 @@ en:
       update_error:
         title: Error
         message: "Something went wrong updating Jurisdiction Customization"
+  services:
+    auto_compliance_configuration:
+      digital_seal_validator:
+        label: "Digital Seal Validator - Validates the digital seal of a file"
+      parcel_info_extractor:
+        label: "Parcel Info Extractor - Extracts info from parcel information: PID, PIN, etc"
+        available_field_labels:
+          feature_area: "Feature area (sqm)"
+          pid: "PID"
```

Comment:
Optional: refactor this to reduce the number of repeated translations

Reply:
@bkmaibach I am going to leave this as it is, because the translations are for different modules. I am intentionally leaving them separate to reduce coupling